bsip-0022.md

```diff
@@ -87,15 +87,15 @@ A voter changes their vote for witnesses on December 31, 2019 20:00 UTC.  This v
 
 ![Qualitative decay of voting power](bsip-0022/general-vote-decay.png)
 
-The voting power of an [account's voting stake](#account-voting-stake) shall be set to full voting power whenever an account's vote is re-cast for _any_ [referendum category](#referendum-categories).  The voting power shall remain at 100% for each of the referendum categories for the entire ["full-power period"](#vote-decay-parameters).
+The voting power of an [account's voting stake](#account-voting-stake) shall be set to full voting power whenever an account's vote is re-cast for _any_ [referendum category](#referendum-categories) or is re-delegated to a voting proxy.  The voting power shall remain at 100% for each of the referendum categories for the entire ["full-power period"](#vote-decay-parameters).  _Retracting the delegation from a voting proxy without also casting a vote does not reset the voting power to 100%_.
```

Comment:
> Retracting the delegation from a voting proxy without also casting a vote does not reset the voting power

Right now it is technically not possible to change/unset the voting proxy without also specifying a (possibly empty) list of votes. This could change in the future, so maybe leave this in.

Reply:
Note that with the proposed explicit voting operation, not changing anything *does* reset the voting power. This is a contradiction.